### DISCUSSION TITLE
Smash Loading Forever

### DISCUSSION THREAD

**Discussion by Dedederp-Potato** [Raytwo/ARCropolis#330] [Issues] [answered]
So I Load Smash After Importing Payload And From What I Saw From Looking Online And See What The Problem Is, And Theres The arcropolis.tomls Thing But I Don't Have It In The Folders, And When I Try And Boot Up Smash It Loads Forever, I Waited 2 Hours And Still No Progress, Anything To Fix This? I Need Help Big Time.

**Comment by Raytwo** [comment, marked as answer]
Sorry But You're Really Not Providing Enough Information Or Context For Us To Help You At The Moment
